client/html/themes/aimeos.js

```diff
@@ -780,7 +788,7 @@ AimeosCatalogFilter = {
 
 		if(aimeosInputComplete.length) {
 			aimeosInputComplete.autocomplete({
-				minLength : 3,
+				minLength : Constants.autocomplete.MIN_LENGTH,
```

Comment:
I like the idea to make this configurable. Can you move that into AimeosCatalogFilter.MIN_INPUT_LEN and leave it to "3" as default? Then, you can simply overwrite it for your project using a line of JS.